miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultWorkstationService.java

```diff
@@ -92,4 +95,17 @@ public ValidationResult validateDeletion(Workstation object) throws IOException
     return result;
   }
 
+  @Override
+  public void validateChange(Workstation workstation, Workstation newWorkstation) throws IOException {
+    List<ValidationError> errors = new ArrayList<>();
+
+    if (newWorkstation == null || !workstation.getAlias().equals(newWorkstation.getAlias())
+        && workstationDao.getByAlias(workstation.getAlias()) != null) {
+      errors.add(new ValidationError("alias", "There is already a workstation with this alias"));
+    }
+
+    if (!errors.isEmpty()) {
+      throw new ValidationException(errors);
+    }
+  }
```

Comment:
Sorry, I had the wrong method name. This is already implemented in the superclass and `collectValidationErrors` is the method that should be overridden here - and it already is.